pub fn main() -> Result<(), std::io::Error>{
    println!("Day 2");
    Ok(())
}